pkg/utils/etcdutil/etcdutil.go

```diff
@@ -41,6 +38,9 @@ const (
 	// defaultEtcdClientTimeout is the default timeout for etcd client.
 	defaultEtcdClientTimeout = 3 * time.Second
 
+	// defaultAutoSyncInterval is the interval to sync etcd cluster.
+	defaultAutoSyncInterval = 60 * time.Second
```

Comment:
BTW, Is there some releationship between the timeouts, like `defaultEtcdClientTimeout` ,`DefaultDialTimeout`,`DefaultRequestTimeout`?